modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/OutputInfoM.scala

```diff
@@ -0,0 +1,20 @@
+package org.ergoplatform.explorer.http.api.v1.models
+
+import org.ergoplatform.explorer.http.api.models.Items
+import derevo.circe.{decoder, encoder}
+import derevo.derive
+import sttp.tapir.{Schema, Validator}
+
+@derive(encoder, decoder)
+final case class OutputInfoM(filteredBoxes: Items[OutputInfo], mempoolBoxes: Items[UOutputInfo])
+
+object OutputInfoM {
+
+  implicit val schema: Schema[OutputInfoM] =
+    Schema
+      .derived[OutputInfoM]
+      .modify(_.filteredBoxes)(_.description("Filtered boxes not spent in mempool"))
```

Comment:
What does "Filtered boxes not spent in mempool" mean?

Reply:
unspent boxes with consideration of the mempool (i.e boxes that have been used in an unconfirmed "UOutput" are filtered out).
I have updated this to return union of all boxes (mempool+non-mempool)